models/load-test.go

```diff
@@ -21,6 +21,9 @@ const (
 
 	// Wrk2LG - represents the wrk2 load generator
 	Wrk2LG LoadGenerator = "wrk2"
+
+	// NighthawkLG - represents the nighthawk load generator
+	NighthawkLG LoadGenerator = "nighthawk"
```

Comment:
Performance preferences for this load generator are currently the same as for the other two load generators, right?

Reply:
I didn't get this, but if it is about the preferences set up from settings then yes they will be same for each load generator.

Reply:
Roger. SGTM